ClothesApp/WebAPI/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,50 @@
+using Application.Dtos.Category;
+using Application.Dtos.SectionCategories;
+using Application.Interfaces.Services;
+using Domain.Entities;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebAPI.Controllers;
+
+[ApiController]
+[Route("api/categories")]
+public class CategoryController : ControllerBase
```

Comment:
How can client get list of categories and navigate through them?

Reply:
Added category tree building